src/app.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta name="description" content="" />
```

Comment:
Do we want to add a description here? It can come in another PR

Reply:
It gets added with the seo component, and that tag would overwrite the one from the seo component, that's why it gets deleted here :D